modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/servlet/taglib/OpenGraphTopHeadDynamicInclude.java

```diff
@@ -333,6 +342,35 @@ private String _addLinkTag(LayoutSEOLink layoutSEOLink) {
 		return sb.toString();
 	}
 
+	private Set<Locale> _getAvailableLocales(
+			Layout layout, long plid, long siteGroupId)
+		throws JSONException {
+
+		Set<Locale> siteAvailableLocales = _language.getAvailableLocales(
+			siteGroupId);
+
+		Set<Locale> availableLocales = new HashSet<>();
+
+		List<FragmentEntryLink> fragmentEntryLinks =
+			_fragmentEntryLinkLocalService.getFragmentEntryLinksByPlid(
+				siteGroupId, plid);
+
+		for (FragmentEntryLink fragmentEntryLink : fragmentEntryLinks) {
+			JSONObject editableValuesJSONObject =
+				JSONFactoryUtil.createJSONObject(
+					fragmentEntryLink.getEditableValues());
+
+			for (String translatableFragment : _TRANSLATABLE_FRAGMENTS) {
+				availableLocales.addAll(
+					_getLocaleOnEditableFragment(
```

Comment:
Can you rename _getLocaleOnEditableFragment to _getEditableFragmentLocale?

Reply:
Or even better, to _getTranslatableFragmentLocale, for consistency with the variable name.